modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -218,7 +218,7 @@ If you are installing a three-node cluster, skip the following step to allow the
 ====
 When you configure control plane nodes from the default unschedulable to schedulable, additional subscriptions are required. This is because control plane nodes then become compute nodes.
 ====
-endif::baremetal,baremetal-restricted,ibm-z,ibm-power[]
+endif::baremetal,baremetal-restricted,ibm-z,ibm-power,aws[]
```

Comment:
@makentenza While the content about skipping the step is valid (line 214), I am wondering if the statement about needing additional subscriptions for AWS is applicable (line 219). Should I keep it or safe to remove?

Reply:
The subscription requirement is independent of the platform. Since user workloads will run in the control plane nodes of a compact cluster, additional subscriptions are required as these control plane nodes are also considered worker/compute nodes.